src/OpenSage.Game/Logic/AI/AIStates/ChinookExitMapState.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace OpenSage.Logic.AI.AIStates;
+
+internal sealed class ChinookExitMapState : State
+{
+    public override void Persist(StatePersister reader) { }
```

Comment:
not even a version?

Reply:
Nope, not even a version. Usually there's a version even when there's no data to persist, but... not always, it seems.